src/App.tsx

```diff
@@ -17,18 +43,18 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter setFilter={setFilter} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loading ? <Loader /> : (
```

Comment:
instead of creating additional state its better to do like Artem said, 

` {filteredTodos.length > 0 ?   <TodoList ... > : <Loader /> `